src/fuselage/CCPACSFuselageSegment.cpp

```diff
@@ -791,4 +799,82 @@ double CCPACSFuselageSegment::GetCircumference(const double eta)
     return System.Mass();
 }
 
+
+void CCPACSFuselageSegment::BuildShell() const
```

Comment:
Where is the difference to BuildLoft? Looks identical to me. The loft includes all three faces, the outer shell and the closure faces. You can just return them instead of recreating them.

In particular, when we change the code that the segments will be lofted in a smooth way over the whole fuselage, this code duplication will result in a different shell than the outer fuselage hull.

Reply:
I agree, but we already have the same behavior on our side. If we use CCPACSFuselage::GetLoft(), the loft is smoothed. Therefore, we have the code which is part of this PR, to get a linearly interpolated loft.
But as I said in the comment below, we might lack some updates from your side.